### ISSUE TITLE
Attribut selector not working anymore

### ISSUE THREAD

**Issue by nlarche** [TylorS/snabbdom-selector#30]
Hello, 

I report some selectors which used to work before v3.0.0:
```
input[disabled] 
input[checked]
option[selected]
```

from 3.0.0 they match only with : 
```
input[disabled=true] 
input[checked=true]
option[selected=true]
```
I don't if they still have to work.

**Comment by atomrc** [comment]
Hey @nlarche, have you tried with the version we deployed this weekend (`v3.1.0`)?

**Comment by nlarche** [comment]
Hi @atomrc, Yes there is the bug with` v3.0.0` and` v3.1.0`

**Comment by jvanbruegge** [comment]
I just released `tree-selector` V2.0.0 with this change

**Comment by atomrc** [comment]
Fix published with version `v4.0.0`
